diff --git a/node_modules/yaml-language-server/.DS_Store b/node_modules/yaml-language-server/.DS_Store
new file mode 100644
index 0000000..151aa34
Binary files /dev/null and b/node_modules/yaml-language-server/.DS_Store differ
diff --git a/node_modules/yaml-language-server/out/server/src/commands.js b/node_modules/yaml-language-server/out/server/src/commands.js
index 3f1643e..74d3a6c 100644
--- a/node_modules/yaml-language-server/out/server/src/commands.js
+++ b/node_modules/yaml-language-server/out/server/src/commands.js
@@ -7,6 +7,6 @@ Object.defineProperty(exports, "__esModule", { value: true });
 exports.YamlCommands = void 0;
 var YamlCommands;
 (function (YamlCommands) {
-    YamlCommands["JUMP_TO_SCHEMA"] = "jumpToSchema";
+    YamlCommands["JUMP_TO_SCHEMA"] = "jumpToArchitectIoSchema";
 })(YamlCommands = exports.YamlCommands || (exports.YamlCommands = {}));
 //# sourceMappingURL=commands.js.map
